### DISCUSSION TITLE
Top and bottom padding for file and folder names

### DISCUSSION THREAD

**Discussion by Mt9ZdS4gw8dREN** [mikekazakov/nimble-commander#264] [Ideas]
To improve the readability of file lists, it would be great to have some basic option to control the height of of a row with padding. At least for me, a bit of space between rows makes the readability greatly improved.

**Comment by mikekazakov** [comment]
This layout information is a table, not a single value, e.g.: https://github.com/mikekazakov/nimble-commander/blob/1a11ddf4dd172f7ac0729f83ae6f40606fe050c4/Source/NimbleCommander/NimbleCommander/States/FilePanels/Brief/PanelBriefView.mm#L32
Right now it's hardcoded, but in theory can be moved into a json-based config if done carefully.
I'd be ok with such change, but it needs to be implemented, so if anybody is interested enough - PRs are welcome.

**Comment by tiagof** [comment]
An attempt at this #551.
Still incomplete as I've exhausted my knowledge of "mac ui development". (see the PR for details)

Would love to have this feature. If anyone can pick it up and finish the PR....

**Comment by tiagof** [comment]
PR was merged. Row height (padding) can now be configured in the Theme.
